spec/mongoid/criteria/queryable/extensions/big_decimal_spec.rb

```diff
@@ -6,55 +6,121 @@
 
   describe ".evolve" do
 
-    context "when provided a big decimal" do
+    context 'when map_big_decimal_to_decimal128 is false' do
```

Comment:
This test needs to either set the option to false or assert that its value is false, to guard against the option value being different due to e.g. a previously executed test.

Reply:
done